### DISCUSSION TITLE
Working with GameObject ?

### DISCUSSION THREAD

**Discussion by ShizuCarry** [nykwil/UnityGGPO#10] [Q&A]
Hello, I'm studying vector wars for quite some days but some question appears.
Can I use an already existing gameobject? It would be useful for using box_colider and animations.
If not, can I take public variables from ParseShipInputs and take them to another script ?
(something like:  public void VwGame(float ParseshipInupts.1.heading) {} )

**Comment by nykwil** [comment]
Unity collision and animation isn't deterministic. It's best to run your game outside of unitys systems. What vector war does is its own game with no unity systems in it operating on plain data. Then before every draw it synchronizes the unity game objects to the data.
